File: Laboratório de Informática 4/Civere13/app/src/main/java/com/example/civere13/models/Comentario.java
package com.example.civere13.models;

public class Comentario {
    public String texto;
    public String utilizador;
    public int classificacao;

    public Comentario(String texto, String utilizador, int classificacao) {
        this.texto = texto;
        this.utilizador = utilizador;
        this.classificacao = classificacao;
    }

    public String getTexto() {
        return texto;
    }

    public String getUtilizador() {
        return utilizador;
    }

    public int getClassificacao() {
        return classificacao;
    }
}

File: Laboratório de Informática 4/Server/Server/Comentario.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Server.Models
{
    public class Comentario
    {
        public String Texto { get; set; }
        public int Utilizador { get; set; }
        public int Votacao { get; set; }
    }
}